### DISCUSSION TITLE
Teams/Webex calendar update

### DISCUSSION THREAD

**Discussion by bestinformatica** [HCL-TECH-SOFTWARE/domino-online-meeting-integration#97] [Q&A]
Is it possible to update Outlook calendar or Webex Meetings list when using DOMI? It seems that when I create a Teams/Webex meeting in Notes using DOMI, Outlook/Webex meetings are not updated.

Could you help me?

Thank you very much

**Comment by paulswithers** [comment]
To update Outlook calendar, you would need to send an invite to that user. I couldn't see a way to map the Teams calendar to anything other than a personal Outlook calendar, and it doesn't pick up groups I'm part of. As a result, I rarely use it.

I'm not sure which Webex meeting list you're referring to. When I create a Webex meeting, it's showing up in the online interface for Webex.
![image](https://user-images.githubusercontent.com/3721367/126963796-fa9211f1-6a03-4839-b40f-977ed8cf4634.png)

**Reply by bestinformatica** [reply]
Thank you Paul, I will try...
